### ISSUE TITLE
support for clair 4.x.y

### ISSUE THREAD

**Issue by piyush94** [goharbor/harbor-scanner-clair#24]
Does the adapter support newer releases of Clair? i.e. 4.x.y

**Comment by danielpacak** [comment]
IIRC Clair v4 has changed the API. Currently this adapter officially supports version v2. Some more tests have to be done to confirm that it works with the newest Clair scanner. (I did a very quick test with Clair v4 some time back but it failed.)

I think we'd appreciate any help on that. The first step could be just to deploy Clair v4 and configure this adapter to use it, and report any issues that may occur during such test. After that we could plan for changes in code.

**Comment by piyush94** [comment]
Yes, in v4 both the Clair API and database relations have changed. Let me see if i can try out the adapter with changes.
